docs/getting-started.md

```diff
@@ -31,13 +34,13 @@
     * In a clustering environment, the `GHE_EXTRA_SSH_OPTS` key must be configured
       with the `-i <abs path to private key>` SSH option.
 
- 3. Add the backup host's SSH key to the GitHub appliance as an *Authorized SSH
+ 3. Add the backup host's (in this case your computer) SSH key to the GitHub appliance as an *Authorized SSH
```

Comment:
I'm not sure I understand this addition. To me, this addition suggests the SSH key the admin user uses to connect to the backup host is the one that needs to be added to the appliance, which isn't necessarily correct.

We encourage customers to use a dedicated host for their backups and if they're using good practices, the backup user will have their own SSH key which won't be the same as the one the admin uses.

Could you explain this addition a bit more so I can understand the background behind its addition.

Reply:
hey @lildude 👋 ! sorry I completely forgot about this PR 🙈

I'm having a hard time remembering what my logic was there and I agree with what you're saying. It doesn't really make sense so I'm removing this line!